languages/clojure/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+A `string` in Clojure is a Java string, which is an object that represents immutable text as a sequence of Unicode characters (letters, digits, punctuation, etc.) and is defined as follows:
+
+```clojure
+(def fruit "Apple")
+```
+
+The `clojure.string` library provides many standard string manipulation and processing functions. In addition, all the usual Java methods for operating on strings are available via interop.
```

Comment:
Do students need to know that the Java methods can be used to solve the exercise? If not, this mention should be removed from this document (it is perfect for an `after.md` document).